bindings/wasm/src/client.rs

```diff
@@ -37,6 +37,12 @@ pub fn create_client(clientOptions: String) -> Result<ClientMethodHandler, JsVal
     Ok(ClientMethodHandler { client })
 }
 
+/// Necessary for compatibility with the node.js bindings.
+#[wasm_bindgen(js_name = destroyClient)]
+pub async fn destroy_client(_client_method_handler: &ClientMethodHandler) -> Result<(), JsValue> {
```

Comment:
So for my understanding, what does this mean exactly? We do not have an implementation for this in Wasm?

Reply:
Yes, in node it's under an `Arc`, but in WASM it's not, so it doesn't require a destroy method.